core/options.c

```diff
@@ -1232,6 +1232,12 @@ static bool
 check_option_compatibility_helper(int recurse_count)
 {
     bool changed_options = false;
+#    if defined(CLIENT_INTERFACE) && defined(AARCH64)
+    if (!DYNAMO_OPTION(bb_prefixes)) {
+        dynamo_options.bb_prefixes = true;
```

Comment:
I think a USAGE_ERROR is called for here to let the user know, like in the other cases.

Reply:
Done.